lib/Console/Traits/Parallelization.php

```diff
@@ -34,6 +34,9 @@ trait Parallelization
         WebmozartParallelization::configureParallelization as parentConfigureParallelization;
     }
 
+    /**
+     * @deprecated Deprecated since webmozarts/console-parallelization 2.0.0 and will be removed in Pimcore 11. Please use configureCommand instead.
```

Comment:
```suggestion
     * @deprecated and will be removed in Pimcore 11. Please use configureCommand instead.
```

Reply:
https://github.com/pimcore/pimcore/pull/13790#discussion_r1048183499 -> this is why I added the webmozarts bundle :)

Reply:
yes, the comment was right since we had webmozart version there `2.0.0` and now we just deprecate the overridden method in pimcore.